File: Medium/maurice_wins.py
def maurice_wins(m_snails, s_snails):
    s_snails=[max(s_snails),min(s_snails),s_snails[1]]
    total =[True if a>b else False for a,b in zip(m_snails,s_snails)]
    return total.count(True)>=2


print(maurice_wins([3, 5, 10], [4, 7, 11]), True)
print(maurice_wins([6, 8, 9], [7, 12, 14]), False)
print(maurice_wins([1, 8, 20], [2, 9, 100]), True)
print(maurice_wins([1, 2, 3], [2, 3, 4]), False)
print(maurice_wins([2, 4, 10], [3, 9, 11]), True)
print(maurice_wins([3, 8, 13], [5, 11, 15]), True)

File: javascript/Easy/mauriceWins.js
// https://edabit.com/challenge/fiLc5x3x2bHzHvgK4



function mauriceWins(mSnails, sSnails) {
	const [ms,mm,mf] = mSnails
    const [ss,sm,sf] = sSnails

    let wins  = 0
    ms>sf?wins+=1:wins;
    mm>ss?wins+=1:wins;
    mf>sm?wins+=1:wins;
    
    return wins>=2
}


console.log(mauriceWins([3, 5, 10], [4, 7, 11]), true);
console.log(mauriceWins([6, 8, 9], [7, 12, 14]), false);
console.log(mauriceWins([1, 8, 20], [2, 9, 100]), true);
console.log(mauriceWins([1, 2, 3], [2, 3, 4]), false);
console.log(mauriceWins([2, 4, 10], [3, 9, 11]), true);
console.log(mauriceWins([3, 8, 13], [5, 11, 15]), true);